examples/WebRadioDemo/WebRadioDemo.ino

```diff
@@ -6,31 +6,38 @@
   
   Wiring:
   --------------------------------
-  | VS1053  | ESP8266 |  Other   |
+  | VS1053  | ESP8266 |  ESP32   |
   --------------------------------
-  |   SCK   |   D5    |    -     |
-  |   MISO  |   D6    |    -     |
-  |   MOSI  |   D7    |    -     |
-  |   XRST  |    -    |    -     |
-  |   CS    |   D1    |    -     |
-  |   DCS   |   D0    |    -     |
-  |   DREQ  |   D3    |    -     |
-  |   5V    |    -    |   VCC    |
-  |   GND   |    -    |   GND    |
+  |   SCK   |   D5    |   IO18   |
+  |   MISO  |   D6    |   IO19   |
+  |   MOSI  |   D7    |   IO23   |
+  |   XRST  |   RST   |   EN     |
+  |   CS    |   D1    |   IO5    |
+  |   DCS   |   D0    |   IO16   |
+  |   DREQ  |   D3    |   IO4    |
+  |   5V    |   5V    |   5V     |
+  |   GND   |   GND   |   GND    |
   --------------------------------
 
   Dependencies:
   -VS1053 library by baldram (https://github.com/baldram/ESP_VS1053_Library)
-  -ESP8266Wifi
+  -ESP8266Wifi/WiFi
 
   To run this example define the platformio.ini as below.
 
   [env:nodemcuv2]
   platform = espressif8266
   board = nodemcuv2
   framework = arduino
-  build_flags = -D PIO_FRAMEWORK_ARDUINO_LWIP2_HIGHER_BANDWIDTH
+  build_flags = -D PIO_FRAMEWORK_ARDUINO_LWIP2_HIGHER_BANDWIDTH -D $PIOENV
+  lib_deps =
+    ESP_VS1053_Library
 
+  [env:esp32dev]
+  platform = espressif32
+  board = esp32dev
+  framework = arduino
+  build_flags = -D PIO_FRAMEWORK_ARDUINO_LWIP2_HIGHER_BANDWIDTH -D $PIOENV
```

Comment:
Thanks for contribution! 
Does it work fine with ESP32? 

I'm not sure this LWIP2 Higher Bandwidth flag is related to this board. 
AFIK it's rather something specific to ESP8266 (please see this Github [search result](https://github.com/search?q=org%3Aplatformio+PIO_FRAMEWORK_ARDUINO_LWIP2_HIGHER_BANDWIDTH&type=Code)).
Maybe some other flag is needed for this variant (or no flag)? But then it would need to be tested whether it works.

Also, when I do this [search in Google](https://www.google.com/search?q=%2BLWIP+esp32+%2Bplatformio), no interesting results. But when you replace "esp32" with "esp8266" in the same query, you immediately [find this](https://docs.platformio.org/en/latest/platforms/espressif8266.html#lwip-variant). So for my eyes it's only esp8266 relevant, but... 
What do you think?

Reply:
I tested with my ESP32.

In ESP32, the same result was obtained with or without the LWIP2 Higher Bandwidth flag.

Reply:
Ok. Then I would go for the variant without the flag.
Will you provide please a change (removing LWIP2 flag for ESP32)?
As I don't see that this build flag is supported by ESP32 platform (I had a look in [spec here](https://docs.platformio.org/en/latest/platforms/espressif32.html)). 
So it would be misleading.